includes/class-wc-payments.php

```diff
@@ -792,16 +801,29 @@ public static function maybe_register_platform_checkout_hooks() {
 	public static function ajax_init_platform_checkout() {
 		$session_cookie_name = apply_filters( 'woocommerce_cookie', 'wp_woocommerce_session_' . COOKIEHASH );
 
+		$user        = wp_get_current_user();
+		$customer_id = self::$customer_service->get_customer_id_by_user_id( $user->ID );
+		if ( null === $customer_id ) {
+			// create customer.
+			$customer_data = WC_Payments_Customer_Service::map_customer_data( null, new WC_Customer( $user->ID ) );
+			self::$customer_service->create_customer_for_user( $user, $customer_data );
```

Comment:
I think the assignation was missed here:

```php
$customer_id = self::$customer_service->create_customer_for_user( $user, $customer_data );
```

ref: p1650474815008989/1650297508.462979-slack-C022WMN88KG

Reply:
Yep, you're definitely right 😬 
Thanks for catching that! 🙏